test/common/testlib.py

```diff
@@ -2080,6 +2080,14 @@ def write_file(self, path: str, content: str, append: bool = False, owner: Optio
         m = self.machine
         self.restore_file(path, post_restore_action=post_restore_action)
         m.write(path, content, append=append, owner=owner, perm=perm)
+    
+    def reboot(self, timeout_sec):
+        self.allow_restart_journal_messages()
+        self.machine.reboot(timeout_sec=timeout_sec)
+
+    def wait_reboot(self, timeout_sec):
+        self.allow_restart_journal_messages()
+        self.machine.wait_reboot(timeout_sec=timeout_sec)
```

Comment:
This is great, but nothing actually uses it. Please adjust the existing tests to use the new `self.wait_reboot`. For example `testTracer` in `test/verify/check-packagekit`.

* Adjust the usage of `m.wait_reboot` in `test/verify/check-packagekit`
* Prepare a test virtual machine: `./test/image-prepare -q fedora-37`
* Run the test: `TEST_OS=fedora-37 ./test/verify/check-packagekit -vst`

If this works as expected (test passes), adjust all other occurrences of `m.wait_reboot` in `test/verify`.